include/aspect/introspection.h

```diff
@@ -106,6 +106,12 @@ namespace aspect
       const bool use_discontinuous_composition_discretization;
 
       /**
+       * A variable that holds whether the composition field(s) should use a
```

Comment:
Currently, I see no place where you would need this variable. Introspection contains some information about the used finite elements, in particular information that need to be used inside of the assembler objects, but this does not seem to be the case for this variable (since you only use it inside of `Simulator`, and you might as well use `parameters.use_limiter_for_discontinuous_solution` there). I see that it might be useful at some point in the future, but as long as we do not need, let us not stuff Introspection full of things that are not necessary right now.